runtime/src/message_processor.rs

```diff
@@ -58,10 +59,36 @@ fn get_subset_unchecked_mut<'a, T>(
         .collect())
 }
 
+pub struct PreAccount {
+    pub debitable: bool,
+    pub lamports: u64,
+    pub data_len: usize,
+    pub data: Option<Vec<u8>>,
+    pub owner: Pubkey,
+    pub executable: bool,
+    pub rent_epoch: Epoch,
+}
+impl PreAccount {
+    pub fn new(a: &Account, debitable: bool, program_id: &Pubkey) -> Self {
+        Self {
+            debitable,
+            lamports: a.lamports,
```

Comment:
is this ok form for us? shortened names to `a.xx`

Reply:
I'd much rather see it spelled out as `account.lamports`.   It makes future diffs much more readable for one.

Reply:
Sometimes having everything spelled out gets noisy, especially in small scopes.  Don't feel strongly either way, will change to more verbose.

Reply:
I'd prefer we only use one-letter variable names in one-line closures, such as:

```rust
fn double(items: &[u64]) -> Vec<u64> {
    items.iter().map(|x| x * 2).collect()
}
```